app/javascript/components/cloud-object-store-container-form/index.jsx

```diff
@@ -43,11 +43,11 @@ const CloudObjectStoreContainerForm = () => {
     }
     miqSparkleOn();
     API.post('/api/cloud_object_store_containers/', submitData).then(() => {
-      const message = __(`Add of Cloud Object Store Container ${values.name} has been successfully queued.`);
+      const message = sprintf(__(`Add of Cloud Object Store Container %s has been successfully queued.`), values.name);
```

Comment:
Think here and in other places we should wrap this in quotes both for consistency and to show the translators that this is the Cloud Object Store Container in question.

```suggestion
      const message = sprintf(__(`Add of Cloud Object Store Container "%s" has been successfully queued.`), values.name);
```